### PR TITLE
Signup changes with private key creation and storage

### PR DESCRIPTION
**Pull request by anoopjkottoor**

### CODE REVIEW

File: src/lib/Popup.svelte [hunk 1]
```diff
@@ -4,12 +4,17 @@
   import Welcome from "./components/popup/Welcome.svelte";
   import Home from "./components/popup/Home.svelte";
   import Logo from "./components/basic/logo.svelte";
-  import { isLoggedIn } from "../lib/store/ui.store";
+  import { isLoggedIn, isSignedUp } from "../lib/store/ui.store";
+  import Singup from "./components/popup/Singup.svelte";
```
Comment (AbrahamGeorge8547) [inline src/lib/Popup.svelte]: signup spelling

File: src/lib/components/popup/Singup.svelte [hunk 2]
```diff
@@ -0,0 +1,183 @@
+<script lang="ts">
+    import browser from "webextension-polyfill";
+    import { writable } from "svelte/store";
+    import { isLoggedIn, isSignedUp } from "../../store/ui.store";
+    import Eye from "../basic/eye.svelte";
+
```
Comment (AbrahamGeorge8547) [inline src/lib/components/popup/Singup.svelte]: use string rather than writable store here.

Comment (AbrahamGeorge8547) [reply src/lib/components/popup/Singup.svelte]: we are already binding the value in input fields.

File: src/lib/components/popup/Singup.svelte [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+                        bind:value={$confirmPassphrase}
+                    />
+                {:else}
+                    <input
+                        class="text-white bg-[#2E3654] border-0 tracking-wider font-normal border-transparent focus:border-transparent focus:ring-0"
+                        type="password"
+                        id="password"
+                        bind:value={$confirmPassphrase}
+                    />
+                {/if}
```
Comment (AbrahamGeorge8547) [inline src/lib/components/popup/Singup.svelte]: use ternary operator here:  type={showPassword ? 'text' : 'password'}

Comment (anoopjkottoor) [reply src/lib/components/popup/Singup.svelte]: this type field cant be made dynamic

File: src/lib/Popup.svelte (hunk 1 above)
Comment (anoopjkottoor) [reply src/lib/Popup.svelte]: Done

**Pull request merged**